exercises/practice/resistor-color/resistor_color_test.gd

```diff
@@ -0,0 +1,23 @@
+func test_color_codes_black(solution_script):
+	return [false, solution_script.color_code("black")]
```

Comment:
The index for "black" should be 0, which is technically falsy, but I think we should compare it with 0 instead ;)

Reply:
Whoops, that's what I get for copying leap's suite over and not double-checking the expected value.